### PR TITLE
GCS to BigTable new template in Java

### PR DESCRIPTION
**Pull request by balajiss2**
#157 

This is a new template for GCS to BigTable in Java. This has capability to load CSV, AVRO and Parquet Files into Bigtable

https://github.com/GoogleCloudPlatform/dataproc-templates/issues/157

### CODE REVIEW

File: java/src/main/resources/template.properties [hunk 1]
```diff
@@ -161,6 +161,18 @@ gcs.bigquery.output.dataset=
 gcs.bigquery.output.table=
 gcs.bigquery.temp.bucket.name=
 
+#GCStoBigTable properties
+# supported formats avro,parquet,csv
+gcs.bigtable.input.format=<csv|parquet|avro>
+#gcs file location
+gcs.bigtable.input.location=<input file location>
```
Comment (vanshaj-bhatia) [inline java/src/main/resources/template.properties:168]: Remove whatever is after =
These values are used for validation. Line 166 is the only correct usage

File: java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigTable.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+    SparkSession spark = null;
+    LOGGER.info("input format: {}", inputFileFormat);
+
+    try {
+      spark = SparkSession.builder().appName("GCS to Bigtable load").getOrCreate();
+
+      Dataset<Row> inputData = null;
+
+      switch (inputFileFormat) {
```
Comment (vanshaj-bhatia) [inline java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigTable.java:68]: Why do we need this switch-case?

Comment (balajiss2) [reply java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigTable.java:68]: The template has the capability to process CSV, AVRO and Parquet files. The switch case is there in order to send the right parameters for the dataframe read operator. For instance, for CSV, we need to provide details on header and all but for parquet,avro there is no need to pass those values. That is the need for the switch case

File: java/src/main/resources/template.properties (hunk 1 above)
Comment (balajiss2) [reply java/src/main/resources/template.properties]: made the changes and committed

File: java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigTable.java [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  }
+
+  @Override
+  public void runTemplate() {
+    validateInput();
+
+    SparkSession spark = null;
+    LOGGER.info("input format: {}", inputFileFormat);
+
+    try {
```
Comment (vanshaj-bhatia) [inline java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigTable.java:63]: remove try-catch

Comment (balajiss2) [reply java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigTable.java]: removed try-catch

**Comment by vanshaj-bhatia** [comment]
LGTM!
Please test the template before merging and proceed with the next steps.

**Pull request merged**